cmd/build-uki.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"os/exec"
+
+	"github.com/kairos-io/enki/pkg/action"
+	"github.com/kairos-io/enki/pkg/config"
+	v1 "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/mount-utils"
+)
+
+// NewBuildISOCmd returns a new instance of the build-iso subcommand and appends it to
+// the root command.
+func NewBuildUKICmd() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "build-uki SourceImage ResultFile KeysDirectory",
+		Short: "Build a UKI artifact from a container image",
+		Long: "Build a UKI artifact from a container image\n\n" +
+			"SourceImage - should be provided as uri in following format <sourceType>:<sourceName>\n" +
+			"    * <sourceType> - might be [\"dir\", \"file\", \"oci\", \"docker\"], as default is \"docker\"\n" +
+			"    * <sourceName> - is path to file or directory, image name with tag version" +
+			"ResultFile - the path to the resulting iso file\n" +
+			"KeysDirectory - the path to the directory with the signing keys.\n" +
+			"    The following files are expected in this directory:\n" +
+			"    - DB.crt\n" +
+			"    - DB.der\n" +
+			"    - DB.key\n" +
+			"    - KEK.der\n" +
+			"    - PK.der\n" +
+			"    - tpm2-pcr-private.pem\n",
+		Args: cobra.ExactArgs(3),
```

Comment:
args should be options flags in some cases IMO.

output -> if not marked just use a generic output.iso
keys -> if not set, just output a uki with no signatures? Should we allow doing that for testing purpouses so people can test before inserting their real keys?